docs/install/java_setup.md

```diff
@@ -35,21 +35,15 @@ The following instructions are provided for macOS and Ubuntu. Windows is not yet
 brew update
 brew tap caskroom/versions
 brew cask install java8
-brew install opencv
```

Comment:
What's going on here?

Reply:
We don't need to install openCV for Java, not any more